src/test/java/elizCurtnazar/tests/tema8/teste/BaseTest.java

```diff
@@ -0,0 +1,41 @@
+package elizCurtnazar.tests.tema8.teste;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import teofilursan.driver.BrowserManager;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BaseTest {
+
+    protected ChromeDriver driver;
+
+    @BeforeMethod(groups = "desktop")
+    public void initChromeBrowser() {
+        driver = new ChromeDriver();
```

Comment:
Replace all of the lines from this method with ->
driver = BrowserManager.createChromeDriverWithOptions();